src/libvirt/Libvirt.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace libvirt
 {
-    public class Libvirt
+    public static class Libvirt
```

Comment:
Better to not limit the capabilities of Libvirt. I would not make it static

Reply:
But there is no non-static member at all? What's your plan about this?